pkgs/top-level/all-packages.nix

```diff
@@ -21618,7 +21618,7 @@ in
     kaddressbook kalzium kapptemplate kapman kate katomic
     kblackbox kblocks kbounce
     kcachegrind kcalc kcharselect kcolorchooser
-    kdenlive kdf kdialog kdiamond
+    kdeconnect-kde kdenlive kdf kdialog kdiamond
```

Comment:
This change will conflict with https://github.com/NixOS/nixpkgs/pull/101369 .

Reply:
Sure, so either this PR or the other needs to be changed depending on which is merged first - there really is no way around that.